src/spikeinterface/sortingcomponents/peak_detection.py

```diff
@@ -631,6 +631,7 @@ def __init__(
         weight_method={},
     ):
         PeakDetector.__init__(self, recording, return_output=True)
+        import scipy
```

Comment:
This is a super meaty import. What if you just import scr_matrix instead?

Reply:
yes, of course, could be clever, I agree... I'll do that asap